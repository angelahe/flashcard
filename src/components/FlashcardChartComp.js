import React from 'react';
import '../styles/flashcard.css';

class FlashcardChartComp extends React.Component {
  constructor() {
    super();
    this.state = {
      show: true,
    };
  }

  render() {
    return (
      <div className="FlashcardChartComp">
        <p>FlashcardChartComp coming soon</p>
      </div>
    );
  }
}
export default FlashcardChartComp;
