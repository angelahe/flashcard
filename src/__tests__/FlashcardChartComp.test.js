import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import FlashcardChartComp from '../components/FlashcardChartComp';

it('renders without crashing', () => {
  shallow(<FlashcardChartComp />);
});

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<FlashcardChartComp />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test('snapshot of UI renders consistently', () => {
  const tree = renderer.create(<FlashcardChartComp />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('contains the container div FlashcardChartComp on shallow render', () => {
  const component = shallow(<FlashcardChartComp />);
  expect(component.html()).toContain('FlashcardChartComp');
});
