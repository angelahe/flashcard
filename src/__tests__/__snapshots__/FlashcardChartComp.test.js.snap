// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of UI renders consistently 1`] = `
<div
  className="FlashcardChartComp"
>
  <p>
    FlashcardChartComp coming soon
  </p>
</div>
`;
